components/frontdoor/variables.tf

```diff
@@ -0,0 +1,40 @@
+variable "env" {
+  type = string
+}
+variable "subscription" {
+  type = string
+}
+variable "project" {
+  type = string
+}
+variable "location" {
+  type = string
+}
+variable "frontends" {
+  type = any
+}
+variable "subscription_id" {
+  type = string
+}
+
+variable "certificate_key_vault_name" {
+  type = string
+}
+variable "oms_env" {
+  type = string
+}
+variable "key_vault_resource_group" {
```

Comment:
This file should only be stored in one place and then symlinked like in azure-platform-terraform, https://github.com/hmcts/azure-platform-terraform/blob/master/components/cftapps_cluster_lb/variables.tf

that's how we get around the `The root module does not declare a variable named "key_vault_resource_group"` warning there